webpack.config.js

```diff
@@ -1,7 +1,7 @@
 /**
  * Webpack configuration file
  **/
-const DashboardPlugin = require('webpack-dashboard/plugin');
+// const DashboardPlugin = require('webpack-dashboard/plugin');
```

Comment:
remove from package.json if not considered a required part of the application.  Users who want to use it should `npm install -g`